cypress/integration/EnrollmentPage/WidgetProfile/index.js

```diff
@@ -0,0 +1,30 @@
+import '../sharedSteps';
+
+beforeEach(() => {
+    cy.loginThroughForm();
+});
+
+Then('the profile details should be displayed', () => {
```

Comment:
Hey, 
Do you think is more sensible to check in this step for the texts fields that are displayed inside the`FlatList` component? 
Thanks!

Reply:
Hi @simonadomnisoru I do think about it but the fields can be different between based on the api version. I think better leave it like this for now and come back after the implementation of edition